components/brave_wallet_ui/page/screens/send/shared.styles.ts

```diff
@@ -36,13 +37,15 @@ export const Text = styled.span<TextProps>`
   --primary: ${leo.color.text.primary};
   --secondary: ${leo.color.text.secondary};
   --tertiary: ${leo.color.text.tertiary};
+  --info: ${leo.color.systemfeedback.infoText};
+  --error: ${leo.color.systemfeedback.errorText};
+  --warning: ${leo.color.systemfeedback.warningText};
```

Comment:
Where are these var used?

Reply:
Is being used in the `Address Message` component 

https://github.com/brave/brave-core/pull/22720/files#diff-01ef48cd87e92556959d11bacfea07208a9217b01585ca80428bf833c3db7182R62

![Screenshot 8](https://github.com/brave/brave-core/assets/40611140/079d0775-a400-4b52-85a6-96545e892c3e)

![Screenshot 9](https://github.com/brave/brave-core/assets/40611140/4006084c-a5a9-47bf-8520-041934c65054)

![Screenshot 11](https://github.com/brave/brave-core/assets/40611140/db3431f5-c19f-4697-9ff0-b5fc5c4d4ea1)